packages/trader/src/Modules/Trading/Containers/chart-widgets.tsx

```diff
@@ -48,16 +47,13 @@ export const DigitsWidget = observer(({ digits, tick }: { digits: TDigits['digit
 });
 
 // Chart widgets passed into SmartCharts
-export const ChartTopWidgets = observer(({ charts_ref, open_market, open }: TChartTopWidgets) => {
+export const ChartTopWidgets = observer(({ open_market, open }: TChartTopWidgets) => {
     const { client, ui } = useStore();
     const { is_digits_widget_active, onChange: onSymbolChange } = useTraderStore();
     const { is_beta_chart } = client;
     const { is_dark_mode_on, is_mobile } = ui;
     const theme = is_dark_mode_on ? 'dark' : 'light';
     let yAxiswidth;
-    if (charts_ref?.chart) {
```

Comment:
`charts_ref` was always undefined because `smartcharts` doesn't accept any ref. Hence, i removed it.